client/.babelrc

```diff
@@ -18,6 +18,7 @@
     }
   },
   "plugins": [
+    "lodash",
```

Comment:
Ooof do we really need lodash? I would rather not add on bulky libraries and depend on ES6.

Reply:
I think we can get rid of this now? If not, we can address this in another PR!